iothub/device/src/TransientFaultHandling/ExponentialBackoff.cs

```diff
@@ -23,55 +23,55 @@
 namespace Microsoft.Azure.Devices.Client.TransientFaultHandling
 {
     /// <summary>
-    /// A retry strategy with backoff parameters for calculating the exponential delay between retries.
+    /// A retry strategy with back-off parameters for calculating the exponential delay between retries.
```

Comment:
hyphenated?

Reply:
my spell checker doesn't like backoff as a single word ... if we all agree that it is a single word I will change these back.

Reply:
compound adjectives should be hyphenated

Reply:
when used as a verb, compound phrases do not use hyphens